src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -211,6 +215,7 @@ def __init__(
             "volume averaged", "nearest neighbor"
         ] = "volume averaged",
         detector_hdir=None,
+        detector_vdir=None,
```

Comment:
Would it make sense to have it be something like `(0, 0, 1)` for `detector_hdir` and `detector_vdir` rather than `None`? Doing that would make it easier for users to see what kind of input is expected and what the vertical & horizontal directions are. It could also simplify some code below.

It'd be best as a `tuple` since mutable default arguments like lists are fraught with peril.

Reply:
The problem is that a constant default value would often give nonsensical results. So, when these are set to None, the code below constructs a reasonable guess based on the provided source and detector points (which are required args)

Reply:
Thanks for the clarification! The way it is now is the better way to do it.